packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/summaryofsteps.md

```diff
@@ -2,6 +2,6 @@ The following diagram shows the sequence of steps for the password recovery use
 
 <div class="common-image-format">
 
-![Displays the password recovery sequence diagram](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-nodejs.png)
+![Sequence diagram displaying the password recovery use case](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-nodejs.png)
```

Comment:
One small nit : In the blue block for step 5, verify should have a capital V. A colon does the same job as a period in this case.